cmd/doctor.go

```diff
@@ -6,93 +6,129 @@ import (
-		* IG?
-	`,
-	DisableAutoGenTag: true,
-	SilenceUsage:      true,
-}
+	platform = &cobra.Command{
+		Use:   "platform",
+		Short: "Verify that operators are installed and ready",
+		Long: `
+		Checks that the ForgeRock platform is running.
+	    `,
+		Example: `
+		# validate the platform is running in the current namespace
+		forgeops doctor platform
+		# validate the platform is running in the "prod" namespace
+		forgeops doctor platform -n prod
+		`,
+		DisableAutoGenTag: true,
+		SilenceUsage:      true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			hlth, err := health.HealthFromBytes(doctor.DefaultPlatformHealth)
+			if err != nil {
+				return err
+			}
+			return health.Run(clientFactory, hlth, false)
+		},
+	}
 
-// Operators
-var ignoreOperators []string
-
-var operators = &cobra.Command{
-	Use:     "operator",
-	Aliases: []string{"op"},
-	Short:   "Verify that operators are installed and ready",
-	Long: `
-	Checks to ensure that required operators are installed and ready.
-	Searches all namespaces for the default deployment of secret agent, nginx-ingress, and cert-manager
-	Checks for a minimum ready count of one.
-	`,
-	SilenceErrors: true,
-	SilenceUsage:  true,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		client, err := clientFactory.StaticClient()
-		if err != nil {
-			return err
-		}
-		err = doctor.CheckOperators(ctx, client)
-		if err != nil {
-			return err
-		}
-		return nil
-	},
-	DisableAutoGenTag: true,
-}
+	operators = &cobra.Command{
+		Use:     "operators",
+		Aliases: []string{"op", "operator"},
+		Short:   "Verify that operators are installed and ready",
+		Long: `
+	    Checks to ensure that required operators are installed and ready.
+	    `,
+		DisableAutoGenTag: true,
+		SilenceErrors:     true,
+		SilenceUsage:      true,
+		Example: `
+		# check for operators in any namespaces
+		forgeops doctor operators
+		# check for operators in single namespace
+		forgeops doctor operators --all-namespaces=false
+		`,
 
-var doctorCmd = &cobra.Command{
-	Use:     "doctor",
-	Aliases: []string{"dr"},
-	Short:   "Diagnose common cluster and platform deployments",
-	Long: `
-	Diagnose common cluster and platform deployments
-    `,
-	// Configure Client Mgr for all subcommands
-	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		clientFactory = factory.NewFactory(doctorFlags)
-	},
-	DisableAutoGenTag: true,
-}
+		// Configure Client Mgr for all subcommands
+		RunE: func(cmd *cobra.Command, args []string) error {
+			hlth, err := health.HealthFromBytes(doctor.DefaultOperatorHealth)
+			if err != nil {
+				return err
+			}
+			return health.Run(clientFactory, hlth, allNamespaces)
+		},
+	}
+
+	doctorCmd = &cobra.Command{
+		Use:               "doctor",
+		Aliases:           []string{"dr"},
+		DisableAutoGenTag: true,
+		Short:             "Diagnose common cluster and platform deployments",
+		Long: `
+		Diagnose issues related to running and deploying the ForgeRock platform.
```

Comment:
Our TMed product name is "ForgeRock Identity Platform". I found "ForgeRock platform" scattered throughout our READMEs when I was tweaking them the last couple of weeks. Could you please fix this here and elsewhere?

You can abbreviate it as "the platform" once you've used the TMed name.